packages/flutter_tools/lib/src/service_protocol.dart

```diff
@@ -8,9 +8,13 @@ import 'device.dart';
 
 /// Discover service protocol ports on devices.
 class ServiceProtocolDiscovery {
-  /// [logReader] A [DeviceLogReader] to look for Observatory messages in.
-  ServiceProtocolDiscovery(DeviceLogReader logReader)
-      : _logReader = logReader {
+  static const String kObservatoryService = 'Observatory';
+  static const String kDiagnosticService = 'Diagnostic server';
```

Comment:
I envision it being used to expose other debugging features.  But there aren't any other specific features planned right now.